js/index.js

```diff
@@ -40,6 +40,26 @@
         });
     }
 
+    window.addEventListener('keydown', function(e) {
```

Comment:
Now it works with InboxView, it was only missing the tabindex=1 attribute for the inbox div to capture the keydown.